single_include/mio/mio.hpp

```diff
@@ -793,6 +793,16 @@ inline DWORD int64_low(int64_t n) noexcept
 {
     return n & 0xffffffff;
 }
+	
+std::wstring s2ws(const std::string& s)
+{
+    if (s.empty()) 
+        return{};
+    const auto sLength = static_cast<int>(s.length());
+    auto buf = std::vector<wchar_t>(sLength);
+    const auto wideCharCount = MultiByteToWideChar(CP_UTF8, 0, s.c_str(), sLength, buf.data(), sLength);
```

Comment:
nitpick: Can the variables here use camel case to be consistent with the rest of the codebase (widnows conventions notwithstanding)?

Reply:
you mean the snake case for 
wide_char_count and s_length?

Reply:
yep